src/test/scala/SystemTests.scala

```diff
@@ -83,7 +83,7 @@ class SystemTests extends AnyFunSuite {
     Logger.info(outPath)
     val timer = PerformanceTimer(s"test $name/$variation")
 
-    val args = mutable.ArrayBuffer("--adt", ADTPath, "--relf", RELFPath, "--output", outPath)
+    val args = mutable.ArrayBuffer("--adt", ADTPath, "--relf", RELFPath, "--output", outPath, "--analyse")
```

Comment:
Do not make this the default for running tests. Make a separate test suite for that.